wizard/src/utils/heuristics.ts

```diff
@@ -27,41 +27,41 @@ import {
 import * as dd from "./detect_date";
 import { PlaceDetector } from "./detect_place";
 
+const STATE_PROPERTY_ORDER = ["isoCode", "fips52AlphaCode", "name"];
+
+const COUNTRY_PROPERTY_ORDER = [
+  "isoCode",
+  "countryAlpha3Code",
+  "countryNumericCode",
+  "name",
+];
+
 /**
- * countryOrder is a helper function which returns the column index of the
- * detected country column based on a ranked order of preferred property types.
+ * placePropertyOrder is a helper function which returns the column index of the
+ * detected place column based on a ranked order of preferred property types.
  * For example, given two columns both of which represent countries, if one of
  * them has ISO codes and the other country numbers, we will prefer the one with
  * ISO codes.
  *
- * The detection order for type:Country is:
- *    1. ISO code
- *    2. Alpha Numberic 3 Letter Abbreviation
- *    3. Numeric code
- *    4. Country name.
- *
- * @param detectedCountries is a mapping from column indices to the DetectedDetails
+ * @param detectedPlaces is a mapping from column indices to the DetectedDetails
  *    objects which contain the specifics of the type and property detected.
  *
  * @returns the column index of the most preferred country column.
  */
-function countryOrder(detectedCountries: Map<number, DetectedDetails>): number {
-  const propOrder = [
-    "isoCode",
-    "countryAlpha3Code",
-    "countryNumericCode",
-    "name",
-  ];
+function placePropertyOrder(
+  detectedPlaces: Map<number, DetectedDetails>,
+  propertyOrder: Array<string>
+): number {
   const propDetected = new Map<string, number>();
 
-  detectedCountries.forEach((details: DetectedDetails, index: number) => {
+  detectedPlaces.forEach((details: DetectedDetails, index: number) => {
     const prop = details.detectedTypeProperty.dcProperty.dcid;
     propDetected.set(prop, index);
   });
 
-  for (let i = 0; i < propOrder.length; i++) {
-    if (propDetected.has(propOrder[i])) {
-      return propDetected.get(propOrder[i]);
+  for (let i = 0; i < propertyOrder.length; i++) {
```

Comment:
nit: can do `for (const prop of propertyOrder) {...}` to access each value in the propertyOrder array